modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb

```diff
@@ -56,6 +57,10 @@ def collection_resource
         end
       end
 
+      def filter_non_va_meds(data)
+        data.reject { |item| item[:prescription_source] == 'NV' && item[:disp_status] != 'Active: Non-VA' }
```

Comment:
Would you be able to cover this logic in the spec?

Reply:
@robertbylight I tried covering this in a spec but the data returned from the API doesn't have any 'invalid' data like we saw on the Veteran's account. Is there a way you can think of that I might be missing?

Reply:
You could try to copy a cassette rename it and hardcode the invalid data.

Reply:
also discussed adding 'invalid' data to our test data returned from the API and he said that wouldn't be an option. So I don't think there is a way to create a valid test for this scenario